pkg/roachprod/vm/azure/azure.go

```diff
@@ -495,6 +495,10 @@ func (p *Provider) DeleteCluster(l *logger.Logger, name string) error {
 		}
 	}
 
+	if len(futures) == 0 {
+		return errors.Newf("DeleteCluster: Found no azure resource groups with tag cluster: %s", name)
```

Comment:
How will this error show up in the slack channel? E.g., will it be lots of flashing red signs, so we won't miss it? :)